test/functional/wallet_send.py

```diff
@@ -33,12 +33,16 @@ def skip_test_if_missing_module(self):
     def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
                   arg_conf_target=None, arg_estimate_mode=None, arg_fee_rate=None,
                   conf_target=None, estimate_mode=None, fee_rate=None, add_to_wallet=None, psbt=None,
-                  inputs=None, add_inputs=None, change_address=None, change_position=None, change_type=None,
+                  inputs=None, add_inputs=None, include_unsafe=None, change_address=None, change_position=None, change_type=None,
                   include_watching=None, locktime=None, lock_unspents=None, replaceable=None, subtract_fee_from_outputs=None,
                   expect_error=None):
         assert (amount is None) != (data is None)
 
-        from_balance_before = from_wallet.getbalance()
+        if include_unsafe:
+            from_balance_before = from_wallet.getbalances()["mine"]["untrusted_pending"]
+        else:
+            from_balance_before = from_wallet.getbalances()["mine"]["trusted"]
```

Comment:
While this works for the purposes of the test, should `from_balance` include "trusted" as well:
```python
        from_balance_before = from_wallet.getbalances()["mine"]["trusted"]
        if include_unsafe:
            from_balance_before += from_wallet.getbalances()["mine"]["untrusted_pending"]
```

Reply:
That makes more sense indeed, done in https://github.com/bitcoin/bitcoin/pull/21359/commits/0709545365d045b24563b4683881f65a2b34afdd